docs/tutorials/qGAN.ipynb

```diff
@@ -0,0 +1,2644 @@
+{
```

Comment:
<p>I need to change this to reflect the fact I removed the layer with 50 nodes. Though I'm not sure if I should add it back as removing it didn't seem to make a difference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/quantum/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>